package.json

```diff
@@ -16,6 +16,7 @@
     "ci:test": "./scripts/ci-test.sh",
     "pretest": "turbo run pretest",
     "test": "jest --no-cache --runInBand --coverage",
+    "test:e2e": "jest --no-cache --runInBand packages/fuel-gauge/src/live-script.test.ts",
```

Comment:
I assume we all have personal derivations of how we call each test type and how each translates to our specific (Fuel) case, and I know @danielbate has been sorting this out and writing some specs around the topic, but I'll leave my current line of thought here, just in case.

 - `Unit` = Worries about units, usually mocking external stuff
 - `Feature` = Worries about features, not mocking or mocking minimally
 - `Integration` = Tests that target `fuel-core` locally
 - `E2E` - Tests that target `fuel-core` remotely

In other words, I am fond of using `e2e` for this case.

Reply:
`test:live` seems like an options as well.